ddtrace/contrib/requests/patch.py

```diff
@@ -39,8 +39,15 @@ def _traced_request_func(func, instance, args, kwargs):
 
     method = kwargs.get('method') or args[0]
     url = kwargs.get('url') or args[1]
+    headers = kwargs.get('headers', {})
 
     with tracer.trace("requests.request", span_type=http.TYPE) as span:
+        if 'x-datadog-trace-id' not in headers:
+            headers['x-datadog-trace-id'] = span.trace_id
```

Comment:
I think we should move such kind of headers in the `ext` package so that it's available in all integrations (and we avoid discrepancy) between them.